sentry/src/main/java/io/sentry/DefaultSentryClientFactory.java

```diff
@@ -428,7 +428,7 @@ protected Connection createHttpConnection(Dsn dsn) {
         httpConnection.setMarshaller(marshaller);
 
         int timeout = getTimeout(dsn);
-        httpConnection.setTimeout(timeout);
+        httpConnection.setConnectionTimeout(timeout);
```

Comment:
One more idea, what if this still called `setTimeout` but `setTimeout` set both of the timeouts?

Right now users have no way to easily increase both timeouts via configuration (without changing Java code). If `setTimeout` was called and set them both, then `getTimeout(dsn)` would continue working as expected.